src/direct_delta_mush.cpp

```diff
@@ -0,0 +1,116 @@
+#include <common.h>
+#include <npe.h>
+#include <typedefs.h>
+#include <igl/direct_delta_mush.h>
+
+const char* ds_direct_delta_mush = R"igl_Qu8mg5v7(
+Perform Direct Delta Mush Skinning. 
+Computes Direct Delta Mesh Skinning (Variant 0) from 
+"Direct Delta Mush Skinning and Variants"
+
+Parameters
+----------
+v  #V by 3 list of rest pose vertex positions
+e  #E Number of bones
+t  #E*4 by 3 list of bone pose transformations
+omega #V by #E*10 list of precomputated matrix values
+
+Returns
+-------
+u  #V by 3 list of output vertex positions
+
+See also
+--------
+
+
+Notes
+-----
+None
+
+Examples
+--------
+)igl_Qu8mg5v7";
+
+npe_function(direct_delta_mush)
+npe_doc(ds_direct_delta_mush)
+
+npe_arg(v, dense_double)
```

Comment:
why only double?

Reply:
If the question is about why direct_delta_mush.cpp (https://github.com/libigl/libigl/blob/main/include/igl/direct_delta_mush.cpp#L267) is only double, I think it's because DDM has some artifacts when using single precision. 

If the question is why double for our python bindings, it is because I felt that it best reflected that DDM only works on double precision. And that work needs to be done for single precision. IMaybe I can let you pick an option here

Option 1. I can leave this as a comment in the file.
Option 2. Take in float as well and cast it.

Reply:
yeah, it looks like the upstream libigl function has Affine3d hard coded. I doubt that double precision is _needed_, just seems like the libigl function should be better templated. I think Option 1 is fine until libigl changes (happy to merge such a PR there).